plugins/processors/emfProcessor/emfProcessor_test.go

```diff
@@ -45,7 +53,7 @@ func buildEMFMetricRule() *structuredlogscommon.MetricRule {
 	return &structuredlogscommon.MetricRule{
 		Namespace:     "ContainerInsights/Prometheus",
 		DimensionSets: [][]string{{"tagA"}},
-		Metrics:       []structuredlogscommon.MetricAttr{{Name: "metric_a"}},
+		Metrics:       []structuredlogscommon.MetricAttr{{Name: "metric_a", Unit: "Count"}},
```

Comment:
You changed the metric rule to add the unit. This changes the existing test. I think it is probably better to keep the original test (without unit). You could add a new test with unit.

Reply:
Good catch. I added back the original test without metric unit support.